src/test/java/org/jenkinsci/plugins/workflow/cps/SnippetizerTest.java

```diff
@@ -125,34 +154,43 @@
         BuildTriggerStep step = new BuildTriggerStep("downstream");
         assertRoundTrip(step, "build 'downstream'");
         step.setParameters(Arrays.asList(new StringParameterValue("branch", "default"), new BooleanParameterValue("correct", true)));
+        // when this test fails because of new core, see below "buildTriggerStepWith_2_2"
```

Comment:
Yes, but we should fix the cases we know about now, and continue to fix others as they become known.